roles/nodejs/molecule/default/verify.yml

```diff
@@ -2,17 +2,34 @@
 - name: Verify
   hosts: all
   gather_facts: false
+
   tasks:
-  - name: check yarn package status
-    ansible.builtin.apt:
-      name: "{{ item }}"
-      state: present
-    check_mode: true
-    register: pkg_status
-    loop:
-      - yarn
+    - name: Check Node.js version
```

Comment:
Yarn is now installed differently. The previous test checked for presence of a apt-get installed yarn which no longer works is not recommended or supported. Since I removed these in the tasks I added new verification of yarn install.